lib/command/Command.js

```diff
@@ -279,17 +279,29 @@ class Command {
         return true;
     }
 
-    process(args, msg) {
+    async process(args, msg) {
```

Comment:
One thing I would like to mention is that async/await isn't a feature that's enabled without a flag until node 7.6.x. You should make sure that what you're adding is compatible with node 4.